t/display.t

```diff
@@ -17,4 +17,13 @@ $lc = 'de';
 is( display_date($t), '27. August 2016 um 12:08:49 CEST' );
 is( display_date_tag($t), '<time datetime="2016-08-27T12:08:49">27. August 2016 um 12:08:49 CEST</time>' );
 
+is(
+	display_field({link => "https://www.brouwerijdebrabandere.be/fr/marques/bavik-super-pils"}, "link"),
```

Comment:
The tests fail, because `display_field` is not exported in https://github.com/openfoodfacts/openfoodfacts-server/blob/a26bf14bd29e1d4d329cf76281ac0ac0d2340862/lib/ProductOpener/Display.pm#L31-L91